build.proj

```diff
@@ -41,11 +24,9 @@
     </Project>
     <!- signing must happen before packaging -!>
     <Project Include="src\sign.builds" />
-    -->
     <Project Include="src\packages.builds" Condition="'$(BuildPackages)'=='true'">
```

Comment:
I added the package build back (although currently it doesn't build any packages because of the traversal work that @chcosta did).